aliases/available/git.aliases.bash

```diff
@@ -66,6 +66,7 @@ alias gftv='git fetch --all --prune --tags --verbose'
 alias gfv='git fetch --all --prune --verbose'
 alias gmu='git fetch origin -v; git fetch upstream -v; git merge upstream/$(get_default_branch)'
 alias gup='git fetch && git rebase'
+alias gfrb="gf && grb origin/$(get_default_branch)"
```

Comment:
Being doubly-opinionated on both remote name and local branch, I'm feeling this may be better-named as `gfrbom` ie `"[g]it [f]etch and [r]ebase [o]rigin against [m](ain|aster)"`

Also using `grb` feels a bit problematic ie. doesn't bash try to pre-resolve the alias, but it hasn't been defined yet (its defined later in the file) - Might just spell out `git rebase` here ...

Reply:
Bash does not try to pre-resolve the alias when defining an alias. It *does* capture aliases for functions.

Reply:
I got it the fix is in plugins, not alias 🤦🏻‍♂️ . My bad

Long time no work on bash has its side-effects

Reply:
```suggestion
alias gfrb='gf && grb origin/$(get_default_branch)'
```